app-emulation/nesalizer/nesalizer-1.0~git.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="SDL-based NES emulator"
+DESCRIPTION="A work-in-progress NES emulator with a real-time rewind
+feature that correctly reverses sound."
+HOMEPAGE="https://github.com/ulfalizer/nesalizer"
+COPYRIGHT="2013-2015 Ulf Magnusson"
+LICENSE="GNU GPL v2"
+REVISION="2"
```

Comment:
REVISION="1"

Reply:
Got confused by this particular [recipe](https://github.com/haikuports/haikuports/blob/c2a214e6473117adfdbf84d0f792341b30e1bbd7/sys-devel/bc/bc-1.07.1.recipe).